### DISCUSSION TITLE
特效的色相/饱和度还原跟AE有差异

### DISCUSSION THREAD

**Discussion by ynot16** [Tencent/libpag#2477] [General]
![image](https://github.com/user-attachments/assets/f594a85b-53ff-453e-a567-4fd1f2af4bc7)

**Comment by kevingpqi123** [comment]
这类问题请提供下 AE 工程，我们好针对性的进行分析

**Reply by ynot16** [reply]
[testHue.zip](https://github.com/user-attachments/files/17053135/testHue.zip)
已上传AE项目

**Reply by kevingpqi123** [reply]
感谢，我们下周看下这个问题

**Reply by ynot16** [reply]
请问这个问题在看了吗

**Comment by CodeJhF** [comment]
之前请你补充的复现信息（demo / 版本号 / 机型 / 日志）一直没有收到，缺少这些信息无法继续定位，先关闭本条。
如果问题还在，欢迎补齐上述信息后回复或重开，我们会优先跟进。（stale-close）
